pkg/ebpf/c/common/buffer.h

```diff
@@ -419,8 +419,13 @@ static __always_inline int save_args_to_submit_buf(event_data_t *event, u64 type
                 size = sizeof(int[2]);
                 rc = save_to_submit_buf(event, (void *) (args->args[i]), size, index);
                 break;
+            case TIMESPEC_T:
+                size = sizeof(struct __kernel_timespec);
+                rc = save_to_submit_buf(event, (void *) (args->args[i]), size, index);
+                break;
         }
-        if ((type != NONE_T) && (type != STR_T) && (type != SOCKADDR_T) && (type != INT_ARR_2_T)) {
+        if ((type != NONE_T) && (type != STR_T) && (type != SOCKADDR_T) && (type != INT_ARR_2_T) &&
```

Comment:
This is getting too big. We should change to:

![image](https://user-images.githubusercontent.com/7395852/220240205-4af954c3-0a17-4c7b-b8d5-08d4c6603e22.png)

So changes here won't change the entire line and are more maintainable.

Reply:
https://github.com/aquasecurity/tracee/pull/2755